src/frontend/app/features/cloud-foundry/add-space/create-space-step/create-space-step.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="add-space">
+  <div>Specify the name of the new space</div>
+  <form [formGroup]="createSpaceForm" class="add-space__stepper-form">
```

Comment:
If you use the class `stepper-form` then the input padding will be uniform with other full page edit steppers